### DISCUSSION TITLE
GitHub Mobile screen flickers a lot when viewing a README with embedded GIF animation

### DISCUSSION THREAD

**Discussion by frak0d** [community/community#50309] [Mobile]
GitHub Mobile screen flickers a lot when viewing a README with embedded GIF animation.
For example try viewing this repo on GitHub mobile app (android) : https://github.com/sigstore/cosign

It appears to be flickering with each frame update in GIF. It's very annoying. If it didn't flicker, try scrolling up and down.

**Comment by Mohamed-SayedAlAhl** [comment]
it seems that the issue with flickering when viewing a README with embedded GIF animations on the GitHub mobile app is a problem with the app's rendering of the animations. There is not much that can be done from the user side to fix this issue, and it would require intervention from GitHub's developers to resolve it. 

In the meantime, you can try accessing the same content on a desktop or laptop computer to see if the issue persists there. Additionally, you can try reaching out to GitHub support or the repository owner to report the issue and see if they can provide any insights or a workaround.

**Comment by frak0d** [comment]
Yes i can see that its a rendering issue. I thought THIS was the place to make github bug reports thats why i posted.
